README.md

```diff
@@ -210,106 +185,91 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
-| Verification              | ❌         | ❌     | ❌   | ❌    |
-| Validation                | ❌         | ❌     | ❌   | ❌    |
-
-### [SD-JWT](https://datatracker.ietf.org/doc/draft-ietf-oauth-selective-disclosure-jwt/) / [SD-JWT-VC](https://datatracker.ietf.org/doc/draft-ietf-oauth-sd-jwt-vc/)
-
-| Feature             | Typescript | Kotlin | Rust | Swift |
-| ------------------- | ---------- | ------ | ---- | ----- |
-| Creation            | ❌         | ❌     | ❌   | ❌    |
-| Signing             | ❌         | ❌     | ❌   | ❌    |
-| Verification        | ❌         | ❌     | ❌   | ❌    |
-| Validation          | ❌         | ❌     | ❌   | ❌    |
-
-### [Bitstring Status List](https://w3c.github.io/vc-bitstring-status-list/)
-
-| Feature                           | Typescript | Kotlin | Rust | Swift |
-| --------------------------------- | ---------- | ------ | ---- | ----- |
-| Creation using `vc-jwt`           | ❌         | ❌     | ❌   | ❌   |
-| Validation using `vc-jwt`         | ❌         | ❌     | ❌   | ❌   |
-
-### [VC JSON Schema](https://www.w3.org/TR/vc-json-schema/)
-
-| Feature                           | Typescript | Kotlin | Rust | Swift |
-| --------------------------------- | ---------- | ------ | ---- | ----- |
-| Creation `JsonSchema`             | ❌         | ❌     | ❌   | ❌   |
-| Creation `JsonSchemaCredential`   | ❌         | ❌     | ❌   | ❌   |
-| Validation `JsonSchema`           | ❌         | ❌     | ❌   | ❌   |
-| Validation `JsonSchemaCredential` | ❌         | ❌     | ❌   | ❌   |
-
-### [Presentation Exchange V2](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
-
-| Feature                | Typescript | Kotlin | Rust | Swift |
-| ---------------------- | ---------- | ------ | ---- | ----- |
-| Concrete Type          | ✅         | ✅     | ❌   | ❌    |
-| Validation             | ✅         | ✅     | ❌   | ❌    |
-| Credential Evaluation  | ✅         | ✅     | ❌   | ❌    |
-| [Predicates](https://identity.foundation/presentation-exchange/spec/v2.0.0/#predicate-feature)                           | ✅         | ✅     | ❌   | ❌    |
-| [Relational Constraints](https://identity.foundation/presentation-exchange/spec/v2.0.0/#relational-constraint-feature)   | ✅         | ❌     | ❌   | ❌    |
-| [Credential Status](https://identity.foundation/presentation-exchange/spec/v2.0.0/#credential-status-constraint-feature) | ❌         | ❌     | ❌   | ❌    |
+| Feature                  |
+| ------------------------ |
+| Data model               | 
+| Signing and verification | 
+
+#### [Status List 2021](https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/)
+
+For usage with the [W3C Verifiable Credential Data Model 1.1](https://www.w3.org/TR/vc-data-model).
+
+| Feature                                | 
+| -------------------------------------- | 
+| Data model                             |
+| Status checking                        |
+| Status setting                         |
```

Comment:
only supporting "Revocation" and "Suspension" ?

Reply:
we will need to support others....is it possible to make it configurable? or would it be better to just list these two for now?

Reply:
hardcoded to 2 for now in both js and kt

Reply:
will specify these two must be supported for now

Reply:
added